doc/content/guides/getting-started/mqtt/_index.md

```diff
@@ -23,7 +23,7 @@ There are many MQTT clients available. Great clients are `mosquitto_pub` and `mo
 >Tip: when using `mosquitto_sub`, pass the `-d` flag to see the topics messages get published on. For example:
 >
 >```bash
-$ mosquitto_sub -h localhost -t '#' -u app1 -P 'NNSXS.VEEBURF3KR77ZR..' -d
+$ mosquitto_sub -h your.thethingsstack.io -t '#' -u app1 -P 'NNSXS.VEEBURF3KR77ZR..' -d
```

Comment:
Not related to this diff; but maybe create the API key in the beginning of this document with the Console instead of CLI?

Reply:
Indeed a bit out of scope for this PR, and then still I think it's fine to assume CLI when they're already using the terminal.

Reply:
They are using the terminal for Mosquitto, not (necessarily) for The Things Stack. I'm fine keeping it here, but then at least add the CLI only shortcode on top of this page

Reply:
Then we should also add that shortcode to other pages that only exaplain how to do things with the CLI and cURL (webhooks, multicast, events), right? I really hoped that we could keep this PR properly scoped 😕